packages/core-components/src/components/button/button.tsx

```diff
@@ -0,0 +1,66 @@
+import { Component, Prop, Element, Event, EventEmitter, h } from '@stencil/core';
+import classNames from 'classnames';
+import { ButtonInterface } from '../../interfaces';
+import { ButtonTypes, ButtonActionTypes, ButtonSizes } from '../../types';
+
+@Component({ tag: 'o-button', styleUrl: 'button.scss', shadow: true })
```

Comment:
[Discussed briefly in my stale PR](https://github.com/okta/odyssey/pull/687/files#r496135166) - but we should adhere to the same prefix for all Odyssey components:

```suggestion
@Component({ tag: 'ods-button', styleUrl: 'button.scss', shadow: true })
```